billcom_integration/models/billcom_item.py

```diff
@@ -0,0 +1,377 @@
+# Copyright 2025 Binhex.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+from .billcom_document import parse_billcom_datetime
+
+_logger = logging.getLogger(__name__)
+
+
+class BillcomItem(models.Model):
+    _name = "billcom.item"
+    _description = "Bill.com Item (Classifications)"
+    _inherit = ["billcom.abstract.model", "mail.thread", "mail.activity.mixin"]
+
+    # Item type - focusing on SALES_TAX for tax mapping
+    type = fields.Selection(
```

Comment:
```suggestion
    item_type = fields.Selection(
```
Avoid using 'type' as a field name - it shadows Python's built-in type() function